test/integration/user-management-api/account_plan_features_test.rb

```diff
@@ -25,52 +26,37 @@ def setup
   end
 
   test 'not found account_plan replies 404' do
-    get(admin_api_account_plan_features_path(0),
-             :provider_key => @provider.api_key, :format => :xml)
+    get admin_api_account_plan_features_path(0), params: { provider_key: @provider.api_key, format: :xml }
     assert_xml_404
   end
 
   test 'enable new feature' do
-    feat = FactoryBot.create :feature, :featurable => @provider
-
-    post(admin_api_account_plan_features_path(@account_plan.id),
-              :feature_id => feat.id,
-              :provider_key => @provider.api_key, :format => :xml)
-
+    post admin_api_account_plan_features_path(@account_plan.id), params: { feature_id: feat.id, provider_key: @provider.api_key, format: :xml }
     assert_response :success
 
     xml = Nokogiri::XML::Document.parse(@response.body)
-
-    assert xml.xpath('.//feature/id').children.first.text == feat.id.to_s
+    assert_equal xml.xpath('.//feature/id').children.first.text, feat.id.to_s
   end
 
   test 'enabling feature not in account replies 404' do
-    feature_not_in_account = FactoryBot.create(:feature,
-                                     :featurable => FactoryBot.create(:provider_account),
-                                     :scope => "AccountPlan")
+    feature_not_in_account = FactoryBot.create(:feature, featurable: FactoryBot.create(:provider_account), scope: "AccountPlan")
 
-    post(admin_api_account_plan_features_path(@account_plan.id),
-              :feature_id => feature_not_in_account.id,
-              :provider_key => @provider.api_key, :format => :xml)
+    post admin_api_account_plan_features_path(@account_plan.id), params: { feature_id: feature_not_in_account.id, provider_key: @provider.api_key, format: :xml }
     assert_xml_404
   end
 
   pending_test 'enable existing feature'
 
   test 'disable feature' do
-    feat = FactoryBot.create :feature, :featurable => @provider
-
-    post(admin_api_account_plan_features_path(@account_plan.id),
-              :feature_id => feat.id,
-              :provider_key => @provider.api_key, :format => :xml)
-
+    post admin_api_account_plan_features_path(@account_plan.id), params: { feature_id: feat.id, provider_key: @provider.api_key, format: :xml }
     assert_response :success
 
     xml = Nokogiri::XML::Document.parse(@response.body)
-
-    assert xml.xpath('.//feature/id').children.first.text == feat.id.to_s
+    assert_equal xml.xpath('.//feature/id').children.first.text, feat.id.to_s
```

Comment:
This seems to be safe here. But just FYI `#assert_equal` is not always safe. See #2627

Reply:
It looks to me that they're exactly the same:
https://github.com/seattlerb/minitest/blob/f4f57afaeb3a11bd0b86ab0757704cb78db96cf4/lib/minitest/assertions.rb#L218